tests/phpunit/tests/post/getAttachedFile.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @group post
+ * @covers ::get_attached_file
+ */
+class Tests_Post_GetAttachedFile extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 36308
+	 */
+	public function test_get_attached_file_with_windows_paths() {
```

Comment:
Using a data provider for the `file`, `expected` values, and possibly the `message` value, will ensure that both assertions run even if one happens to fail, and makes it easier to add more tests for this in future if needed.